services/server/server_test.go

```diff
@@ -188,8 +188,22 @@ func TestServer(t *testing.T) {
 			RemoveAlertManagerRules: true,
 		}))
 
-		s.envSettings.DisableTelemetry = true
+		s.envSettings.EnableUpdates = true
+		expected = status.New(codes.FailedPrecondition, "Updates are enabled via ENABLE_UPDATES environment variable.")
```

Comment:
Do we have `ENABLE_UPDATES` env variable? As far as I know we have only DISABLE_UPDATES

Reply:
let's rewrite test taking into account that we can't enable updates if it's disabled.